tasks/eth/031-op-mainnet-pectra-defense/README.md

```diff
@@ -0,0 +1,47 @@
+# ProxyAdminOwner - Update Prestate Hash
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+This task deploys new dispute games with updated prestate hashes on OP Sepolia.
+
+This updated prestate hash corresponds with the hardfork to support the L1 Pectra upgrade.
+
+## Input data derivation:
+
+The `OPPrestateUpdater` contract was deployed in advance to [0x6E34d06bA5FcA20269036E7c6F3Bf3f774a9A8a6](https://etherscan.io/address/0x6e34d06ba5fca20269036e7c6f3bf3f774a9a8a6).
+
+The `StandardValidator180` contract was deployed in advance to [0xFDa93f2Db6676541dDE71165681d0f23B25163dC](https://etherscan.io/address/0xFDa93f2Db6676541dDE71165681d0f23B25163dC).
```

Comment:
Are these addresses attested to somewhere, even if just a discord message, so the signers know these are correct?

Reply:
They are attested to in this document and signed by might pgp key... I could post them in signal too if you think it makes a difference, but not everyone is in discord.

Reply:
This is probably ok for now, we'll see if anyone asks